### ISSUE TITLE
upperCase/LowerCase is not handled for conflicting claims feature

### ISSUE THREAD

**Issue by prabalsingh24** [coding-blocks/boss#408]
See this comment for more details
https://github.com/coding-blocks/boss/pull/357#pullrequestreview-426794508

**Comment by Ishaan29** [comment]
@hereisnaman can I work on this issue ??

**Comment by aditya237** [comment]
can i work on this issue

**Comment by aditya237** [comment]
> @hereisnaman can I work on this issue ??

heyy is it  your first time

**Comment by Ishaan29** [comment]
> > @hereisnaman can I work on this issue ??
> 
> 
> 
> heyy is it  your first time
> 
> 

Yes, I am new to this repo, but I think this would be a good task to understand the codebase😀

**Comment by aditya237** [comment]
Same here just wanna know

**Comment by deprov447** [comment]
@hereisnaman I would like to work on this issue
